nsv13/code/modules/clothing/custom_clothes.dm

```diff
@@ -23,7 +23,7 @@
 	icon_state = "peacekeeper"
 	worn_icon_state = "peacekeeper"
 	item_state = "bl_suit"
-	armor = list("melee" = 10, "bullet" = 10, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)
+	armor = list("melee" = 20, "bullet" = 20, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)
```

Comment:
```suggestion
	armor = list("melee" = 10, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30, "stamina" = 30)
```
This is just a sec uniform, it shouldn't really have higher stats than the normal one (and also you forgot stamina)

Reply:
Bokkie? Seccie suits have 2 melee, 2 bullet protection.